src/schema/v2/artworksForUser/artworksForUser.ts

```diff
@@ -29,11 +29,13 @@ export const artworksForUser: GraphQLFieldConfig<void, ResolverContext> = {
     version: { type: GraphQLString },
     maxWorksPerArtist: { type: GraphQLInt },
     marketable: { type: GraphQLBoolean },
+    includeAuctionArtworks: {
+      type: new GraphQLNonNull(GraphQLBoolean),
+      defaultValue: false,
+    },
   }),
   resolve: async (_root, args: CursorPageable, context) => {
-    if (!context.artworksLoader) return
```

Comment:
This loader should always exist (it's part of the unauthenticated loaders), so I'm not sure why this guard + spec was around - I think it's over-cautious.